internal/fs/fs.go

```diff
@@ -12,117 +12,103 @@ import (
 
 type CloudStashFs struct {
 	manager *manager.Manager
-
 	fuse.DefaultFileSystem
 }
 
 func NewCloudStashFs(m *manager.Manager) *CloudStashFs {
 	return &CloudStashFs{manager: m}
 }
 
-func (r *CloudStashFs) StatFs(ino int64) (*fuse.StatVFS, fuse.Status) {
+func (fs *CloudStashFs) StatFs(ino int64) (*fuse.StatVFS, fuse.Status) {
 	return nil, fuse.ENOSYS
 }
 
-func (r *CloudStashFs) GetAttr(ino int64, info *fuse.FileInfo) (*fuse.InoAttr, fuse.Status) {
+func (fs *CloudStashFs) GetAttr(ino int64, info *fuse.FileInfo) (*fuse.InoAttr, fuse.Status) {
 	fmt.Printf("getattr ino: %d\n", ino)
 
-	md, err := r.manager.GetMetadata(ino)
+	md, err := fs.manager.GetMetadata(ino)
 	if err != nil {
 		if err == common.ErrNotFound {
 			return nil, fuse.ENOENT
 		}
 
 		fmt.Fprintf(os.Stderr, "couldn't get metadata of inode %d: %v\n", ino, err)
-
 		return nil, fuse.EIO
 	}
 
-	inode := newInode(md)
-
-	return inode, fuse.OK
+	return newInode(md), fuse.OK
 }
 
-func (r *CloudStashFs) SetAttr(ino int64, attr *fuse.InoAttr, mask fuse.SetAttrMask, fi *fuse.FileInfo) (*fuse.InoAttr, fuse.Status) {
+func (fs *CloudStashFs) SetAttr(ino int64, attr *fuse.InoAttr, mask fuse.SetAttrMask, fi *fuse.FileInfo) (*fuse.InoAttr, fuse.Status) {
 	fmt.Printf("setattr ino: %d\n", ino)
 
-	md, err := r.manager.GetMetadata(ino)
+	md, err := fs.manager.GetMetadata(ino)
 	if err != nil {
 		if err == common.ErrNotFound {
 			return nil, fuse.ENOENT
 		}
 
 		fmt.Fprintf(os.Stderr, "couldn't get metadata of inode %d: %v\n", ino, err)
```

Comment:
@paddlesteamer Question; are these `fmt.Printf` and `fmt.Fprintf` statements are used/helpful in any way to the app or are they just for debugging?